src/modules/comments.js

```diff
@@ -0,0 +1,36 @@
+import { capstoneApiURL } from './url.js';
+import createApp from './app.js';
+
+export const getComments = async (movieId) => {
+  const appId = await createApp();
+  try {
+    const response = await fetch(
+      `${capstoneApiURL}/${appId}/comments?item_id=${movieId}`,
+      {
+        method: 'GET',
+      },
+    );
+    const data = await response.json();
+    return data;
+  } catch (error) {
+    throw new Error('Cannot fetch comments.');
+  }
+};
+
+export const giveComments = async ({ movieId, name, comments }) => {
+  const appId = await createApp();
+  console.log(`movieid: ${movieId}:Name:${name}:Comments:${comments}`);
+  const response = await fetch(`${capstoneApiURL}/${appId}/comments`, {
+    method: 'POST',
+    headers: {
+      'Content-Type': 'application/json',
+    },
+    body: JSON.stringify({
+      item_id: movieId,
+      username: name,
+      comment: comments,
+    }),
+  });
+  const data = await response.json();
+  return data;
+};
```

Comment:
- [ ] [Optional] While adding a new comment the comment counter didn't update, Please fix that.

![chrome-capture-2023-2-26 (1)](https://user-images.githubusercontent.com/68796232/227747177-46c2828b-22c8-4526-b1f1-b58c7c00550f.gif)

#### ⚠️I marked this as optional because the last reviewer forgot to mention it so please consider doing it⚠️